### PR TITLE
FIX: Validate interpolation keys used in translation overrides.

### PR DESCRIPTION
**Pull request by tgxworld**
https://meta.discourse.org/t/discobot-translation-missing-error/64429/6?u=tgxworld

### CODE REVIEW

File: config/locales/server.en.yml [hunk 1]
```diff
@@ -409,6 +409,11 @@ en:
           attributes:
             execute_at:
               in_the_past: "must be in the future."
+        translation_overrides:
+          attributes:
+            value:
+              invalid_interpolation_keys: 'The following interpolation key(s) are not : %{keys}'
```
Comment (ZogStriP) [inline config/locales/server.en.yml]: not what? not valid? invalid? not used?

Comment (tgxworld) [inline config/locales/server.en.yml]: Ah good catch!

**Comment by tgxworld** [comment]
Merged in https://github.com/discourse/discourse/commit/b5ec241716b6fb09598e9e82080a3b5a2f645473

**Comment by tgxworld** [comment]
Thanks for reviewing @ZogStriP

**Pull request closed without merging**